app/config.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
   loggerLevel: process.env.LOGGER_LEVEL || 'error',
 
   // The base uri of identity service
-  oidcBaseUri: process.env.OIDC_BASE_URI || 'http://localhost:5102',
+  oidcBaseUri: process.env.OIDC_BASE_URI || 'http://localhost:5102/identity',
```

Comment:
Think we should get rid of this one and just use the `identityServerUrl`.

Reply:
Sam nd I discussed it and decided to keep it separate for now incase it separates later